helm/conjur-app-deploy/charts/app-secrets-provider-rotation/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: app-secrets-provider-rotation
+home: https://www.conjur.org
+version: 0.1.0
```

Comment:
This doesn't need to be included in this PR, but I'm wondering if there is a better approach to these app deployment Helm charts. Instead of making new sub-charts for each deployment case, should we generalize the app deployment charts, specifically for Secrets Provider?

Instead of:
- `app-secrets-provider-init`
- `app-secrets-provider-init-jwt`
- `app-secrets-provider-p2f`
- `app-secrets-provider-p2f-jwt`
- `app-secrets-provider-rotation`
- `app-secrets-provider-standalone` (this might be the exception)

Maybe we could have one chart, `app-secrets-provider`, that uses certain flags to configure either K8s Secrets config, P2F annotations, or JWT auth (or some combo of those).

The singular Helm chart might be more difficult to maintain, but it might be preferable to a new Helm chart for each added test scenario.